Sources/Testing/Attachments/Attachment.swift

```diff
@@ -37,8 +37,7 @@ public struct Attachment<AttachableValue>: ~Copyable where AttachableValue: Atta
   public var fileSystemPath: String?
 
   /// The default preferred name to use if the developer does not supply one.
-  @_spi(ForSwiftTestingOnly)
-  public static var defaultPreferredName: String {
+  private static var defaultPreferredName: String {
```

Comment:
```suggestion
  package static var defaultPreferredName: String {
```

Reply:
There's nothing in other targets which are currently referencing this. Do we actually need it to be `package`?

Reply:
Yes, I have a blocked PR that needs it.